activerecord/lib/active_record/relation/calculations.rb

```diff
@@ -478,7 +487,15 @@ def possible_aggregation?(column_names)
       end
 
       def operation_over_aggregate_column(column, operation, distinct)
-        operation == "count" ? column.count(distinct) : column.public_send(operation)
+        if operation == "count"
+          if column.is_a?(Array)
+            Arel::Nodes::Count.new(column, distinct)
+          else
+            column.count(distinct)
+          end
+        else
+          column.public_send(operation)
```

Comment:
I don't know whether it is possible to have an array of columns for non-count operations. I guess in case that is possible, it would be reported and dealt with in another PR.

But it would be nice if you know the answer off the top of your head :)